recipes/busybox/busybox-1.24.1/defconfig

```diff
@@ -263,16 +289,15 @@ CONFIG_FEATURE_FLOAT_SLEEP=y
 # CONFIG_FEATURE_STAT_FORMAT is not set
 CONFIG_STTY=y
 # CONFIG_SUM is not set
-CONFIG_SYNC=y
```

Comment:
Oh, additional comment to the strange CONFIG_SYNC line above.  It should stay enabled :)